examples/07-TwinBuilder/01-RC_Circuit_Example.py

```diff
@@ -93,32 +90,27 @@
 
 ################################################################
 # Get Report Data and plot it on matplotlib.
-#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Make the line longer as the title above to avoid Sphinx warnings.

Reply:
@maxcapodi78 done